.gitignore

```diff
@@ -29,3 +29,5 @@ fh-log
 integration-test.sh
 public/css/styles.css
 bower_components/
+public/lib/patternfly
```

Comment:
That's not going to work as this is already in the source code.. See separate PR for solution